app/scripts/controllers/eventDetails.js

```diff
@@ -32,14 +32,9 @@ angular.module('confRegistrationWebApp')
     };
     $scope.changeTab($scope.tabs[0]);
 
+    //AUTHORIZE_NET no longer used.
+    //TSYS is now the only payment allowed.
```

Comment:
I don't think these comments will continue to be useful. Seems like it's commenting on code that won't exist in the future. I think you can just toss them. TSYS is the only payment gateway listed below now.